src/commands/ReminderCommand.ts

```diff
@@ -188,33 +246,50 @@ class ReminderCommand extends Command {
 
 const setMinutes = new SlashCommandIntegerOption()
 	.setName("minutes")
-	.setDescription("Set the minutes.")
+	.setDescription("Set the minutes. (60 seconds)")
 	.setMinValue(0)
-	.setMaxValue(60)
+	.setMaxValue(1440)
 	.setRequired(false);
 
 const setHours = new SlashCommandIntegerOption()
 	.setName("hours")
-	.setDescription("Set the hours.")
+	.setDescription("Set the hours. (60 minutes)")
 	.setMinValue(0)
-	.setMaxValue(24)
+	.setMaxValue(720)
 	.setRequired(false);
 
 const setDays = new SlashCommandIntegerOption()
 	.setName("days")
-	.setDescription("Set the days.")
+	.setDescription("Set the days. (24 hours)")
+	.setMinValue(0)
+	.setMaxValue(365)
+	.setRequired(false);
+
+const setMonths = new SlashCommandIntegerOption()
+	.setName("months")
+	.setDescription("Set the months. (30 days)")
 	.setMinValue(0)
-	.setMaxValue(23)
+	.setMaxValue(24)
+	.setRequired(false);
+
+const setDateIso = new SlashCommandStringOption()
+	.setName("date-iso")
+	.setDescription("Set the absolute date in the ISO 8601 format. (e.g. '26 May 2003 04:48:33 UTC+2')")
```

Comment:
iso 8601 is actually `YYYY-MM-DDTHH:mm:ss.sssZ` other formats are not officially supported

Reply:
I know. 
```suggestion
	.setDescription("Set the absolute date in the ISO 8601 format. (e.g. '2003-05-26T04:48:33+02' but others may also work: '26 May 2003 04:48:33 UTC+2')")
```